go.mod

```diff
@@ -1,17 +1,17 @@
 module sigs.k8s.io/apiserver-network-proxy
 
-go 1.12
+go 1.16
 
 require (
 	github.com/golang/mock v1.4.4
 	github.com/golang/protobuf v1.4.2
-	github.com/google/uuid v1.1.1
+	github.com/google/uuid v1.1.2
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/prometheus/client_golang v1.7.1
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/pflag v1.0.5
 	golang.org/x/net v0.0.0-20201110031124-69a78807bb2b
-	google.golang.org/grpc v1.27.0
```

Comment:
Not against upgrading grpc (I tested this myself a few weeks ago) but I would rather we separated this upgrade from the golang upgrade.

Reply:
great point I agree

Reply:
done